org.tlauncher.TLauncher.metainfo.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 TLauncher Inc. -->
+<component type="desktop-application">
+  <id>org.tlauncher.TLauncher</id>
+  <name>TLauncher</name>
+  <summary>The launcher for Minecraft</summary>
```

Comment:
Summary could be better.

Reply:
Not sure how to make it better. The Twitter page states "The popular Minecraft launcher". I guess a compromise would be "A popular launcher for Minecraft"? Not sure what else to put inside.

Reply:
Whatever to indicate it's not official launcher.